### PR TITLE
Performed some correction in the documentation.

### PR DESCRIPTION
**Pull request by sbrunner**
Fix: #835
Replace #851

### CODE REVIEW

File: contribs/gmf/src/directives/mobiledisplayqueries.js [hunk 1]
```diff
@@ -25,13 +25,11 @@ ngeo.module.value('gmfMobiledisplayqueriesTemplateUrl',
 
 
 /**
- * Provide a "gmf mobile display queries" directive.
+ * Provide a directive to displays results of the {@link ngeo.queryResult}
```
Comment (fgravin) [inline contribs/gmf/src/directives/mobiledisplayqueries.js]: display

File: src/directives/control.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ goog.require('ol.control.Control');
 
 
 /**
- * Provides a directive can be used to add a control to a DOM
- * element of the HTML page.
+ * Provides a directive can be used to add a control to the map
```
Comment (fgravin) [inline src/directives/control.js]: `that` can be used

**Comment by fgravin** [comment]
Some typos.
Otherwise LG

**Comment by sbrunner** [comment]
Thanks, fixed, I will merge when Travis is green

**Pull request merged**